components/dayspan-custom-calendar.vue

```diff
@@ -712,6 +722,14 @@ export default {
 
 <style lang="scss">
 
+.noWeekend .ds-day:nth-child(7){
```

Comment:
Für CSS-Klassen nimmt man in der Regel kebab case (populärer Standard zum Beispiel http://getbem.com/introduction/).

Reply:
geändert